python/nano/tutorial/notebook/training/pytorch/accelerate_pytorch_training_multi_instance.ipynb

```diff
@@ -0,0 +1,377 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[View the runnable example on GitHub](https://github.com/intel-analytics/BigDL/tree/main/python/nano/tutorial/notebook/training/pytorch/accelerate_pytorch_training_multi_instance.ipynb)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Accelerate PyTorch Training using Multiple Instances"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`TorchNano` (`bigdl.nano.pytorch.TorchNano`) supports multi-instance training that can make the most of hardwares with multiple CPU cores or sockets (especially when the number of cores is large). Here we provide __2__ ways to achieve this: A) subclass `TorchNano` or B) use `@nano` decorator. You can choose the appropriate one depending on your (preferred) code structure."
```

Comment:
It seems like that this sentence is not complete:

`TorchNano` (`bigdl.nano.pytorch.TorchNano`) supports multi-instance training that can make the most of hardwares with multiple CPU cores or sockets (especially when the number of cores is large) ...?

Do you mean ".... training that can benefit from the most hardwares ..." ?

Reply:
I guess the correct representative might be "that can make **full usage** of hardwires with ..."

Reply:
Here I used __make the most of__ as a phrase equivalent to __make full usage of / take full advantage of__. I will replace it to disambiguate.